typedb/common/streamer.py

```diff
@@ -18,28 +18,20 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-from abc import ABC, abstractmethod
-from typing import TypeVar, Generic, Callable
 
-T = TypeVar('T')
-U = TypeVar('U')
+from typing import Callable
 
 
-class QueryFuture(Generic[T], ABC):
+class Streamer:
```

Comment:
Let's make this generic `Streamer(Generic[T])` returns `T` instances from `__next__()`

Reply:
We decided in-person that we wouldn't like to have a generic for native types.